index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+  </head>
+
+  <body>
+
+    <nav>
+      <ul>
+        <li><a href="#welcome-section"> Welcome </a></li>
+        <li><a href="#works">Work</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
+
+
+    <section class="welcome-section">
+      <h1>Welcome to my portfolio site</h1>
+    </section>
+
+    <section class="works">
+      <h2>Our works</h2>
+      <div class="projects">
+        <div class="project">
+          <div class="project-tile">First Project</div>
+          <a href="https://www.google.com">Link to my project</a>
+        </div>
+        <div class="project">
+          <div class="project-tile">second Project</div>
+          <a href="https://www.google.com">Link to my project</a>
+        </div>
+        <div class="project">
+          <div class="project-tile">third Project</div>
+          <a href="https://www.google.com">Link to my project</a>
+        </div>
+      </div>
```

Comment:
> > I think it would be better to use * and inside a  tag.
> 
> * [ ]  Please always try to explain the reasoning for your suggestion. it may not be obvious to the reader why you are making this suggestion.
> 
> > Your implementation using div tags.
> 
> * [ ]  Also try to be more clear in your bullet points. here It's not clear to me what you mean when you tell them "your implementation using divs" but link to some code that uses shows `ul`s. Please try clearly explain your suggestions. Something that is clear in your own mind might not be so obvious to others.



> > I think it would be better to use * and inside a  tag.
> 
> * [ ]  Please always try to explain the reasoning for your suggestion. it may not be obvious to the reader why you are making this suggestion.
> 
> > Your implementation using div tags.
> 
> * [ ]  Also try to be more clear in your bullet points. here It's not clear to me what you mean when you tell them "your implementation using divs" but link to some code that uses shows `ul`s. Please try clearly explain your suggestions. Something that is clear in your own mind might not be so obvious to others.